### ISSUE TITLE
Customizable InfluxDB2 export interval

### ISSUE THREAD

**Issue by TomeMD** [nicolargo/glances#2348]
**Is your feature request related to a problem? Please describe.**
I want to export cpu metrics to an InfluxDB Server every 2s, but the default value is set to 10000ms (10s) and and cannot be modified.

**Describe the solution you'd like**
Modify InfluxDB export module (https://github.com/nicolargo/glances/blob/master/glances/exports/glances_influxdb2.py) to use a new configuration key "export_interval" and init the write api client (lines 71 to 82) with these value (or 10000ms by default).

**Comment by nicolargo** [comment]
Hi @TomeMD 

Glances exports stats every 'refresh' seconds (refresh can be configured from the command line (-t 2) or from the Glances configuration file https://github.com/nicolargo/glances/blob/develop/conf/glances.conf#L9)

In order to optimize the writing process, the InfluxDB client library store the stats in a batch and write it every 10 seconds into the InfluxDB database: https://github.com/influxdata/influxdb-client-python#batching 

The flush_interval defines the number of milliseconds (10 seconds) before the batch is written by the InfluxDB Client library (batch size is 500).

To summarize, by default, Glances exports stats every 2 seconds (you should have a point every 2 seconds in the InfluxDB database).

**Comment by TomeMD** [comment]
Thanks for your response!

In my case I am exporting metrics from a Glances container, to which I specify --time 2 (although not necessary). I can see that Glances collects metrics every 2 seconds, however when I look at my InfluxDB database I only see data points every 10 seconds.

Here are two screenshots obtained at the same time, the first one shows Glances exporting metrics to InfluxDB and at the same time displaying the cpu.user measurement on screen. The second one shows this measure at my InfluxDB database, where you can see that it only gets data points every 10 seconds. In fact, it matches that every 5 metrics seen in Glances (2s) one is seen in InfluxDB (the fifth, every 10s).
![glances](https://user-images.githubusercontent.com/72162530/233074341-548cc280-7373-4a0a-b7a5-46fffb5324d2.png)
![influxdb](https://user-images.githubusercontent.com/72162530/233074359-2e504125-084d-4ce1-92bc-56aefd42ea76.png)

**Comment by nicolargo** [comment]
Can you please send us the Glances log file ?

**Comment by nicolargo** [comment]
You can also run Glances in debug mode (-d) and check in the Glances log file that export is done every 2 seconds.

**Comment by TomeMD** [comment]
Of course, I am running Glances in a cluster and do not have permissions to access the log files, but I reproduced this scenario in a local container (where the problem persists).

Here is the log file, it seems that the metrics are exported correctly every 2s to InfluxDB (at least the CPU metrics that I am interested in), but in my server I only see this:

![image](https://user-images.githubusercontent.com/72162530/233085189-89fad0fa-f946-4cc0-bbd6-a5366954a2c6.png)

[glances.log](https://github.com/nicolargo/glances/files/11273402/glances.log)

**Comment by nicolargo** [comment]
Can you try to "force" a sampling period to 2 seconds in your query ?

    |> window(every: 2s)

**Comment by TomeMD** [comment]
Yes, here you have the query for the same time range forcing a sampling period of 2s:

![image](https://user-images.githubusercontent.com/72162530/233112847-9498bbcf-61c2-448c-8735-3da51f587b05.png)

It seems to change only the start (`_start`) and end (`_stop`) values of each row.

**Comment by TomeMD** [comment]
Setting the flush_interval to 2000 seems to work, now my InfluxDB database receive data points every 1-3s (because of the jitter I guess):

![flush_interval_2000](https://user-images.githubusercontent.com/72162530/233124041-a56698e6-1c5f-4b17-8786-0719b2a8b7ca.png)

Because of that, it would be nice to be able to modify this parameter directly from the Glances configuration file.

**Comment by nicolargo** [comment]
Done on the develop branch:
- Export interval is set by default to the Glances refresh value (2 seconds by default)
- Behavior can be overwrite using the interval configuration key (in [influxdb2] section)  

Thanks for the issue report @TomeMD !
